package.json

```diff
@@ -1,16 +1,20 @@
 {
   "name": "d2",
-  "version": "0.3.0",
+  "displayName": "d2",
```

Comment:
Should be "D2".

Reply:
Changed.  But... Can I get some guidance from Terrastruct on proper branding?  Proper casing of names, colors, etc...